src/features/session_replay/shared/stylesheet-evaluator.js

```diff
@@ -2,7 +2,7 @@ import { originals } from '../../../common/config/config'
 import { isBrowserScope } from '../../../common/constants/runtime'
 
 class StylesheetEvaluator {
-  #evaluated = new WeakSet()
+  #evaluated = new Set()
```

Comment:
Let's keep this a WeakSet for now to limit the memory footprint of the agent.

```suggestion
  #evaluated = new WeakSet()
```